src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -42,30 +43,28 @@ public void adsRecoveryPageFairOasis(Page page) {
   }
 
   @Test(
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsRecoveryPageFairOasis",
       groups = "AdsRecoveryNoAdblockPageFairOasis"
   )
-  public void adsRecoveryNoAdblockPageFairOasis(Page page) {
-    String url = urlBuilder.getUrlForPage(page);
+  public void adsRecoveryNoAdblockPageFairOasis() {
+    String url = urlBuilder.getUrlForPath(WIKIA, articlePageFairRecoveryUrl());
     AdsRecoveryObject adsRecoveryObject = new AdsRecoveryObject(driver, url, DESKTOP_SIZE);
-    adsRecoveryObject.refreshPageAddingCacheBuster();
 
     adsRecoveryObject.verifyNumberOfPageFairRecoveredSlots(4);
   }
 
   @Test(
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsRecoveryPageFairOasis",
       groups = "AdsRecoveryLoggedInPageFairOasis"
   )
   @Execute(asUser = User.USER_2)
-  public void adsRecoveryLoggedInPageFairOasis(Page page) {
-    String url = urlBuilder.getUrlForPage(page);
+  public void adsRecoveryLoggedInPageFairOasis() {
+    String url = urlBuilder.getUrlForPath(WIKIA, articlePageFairRecoveryUrl());
     AdsRecoveryObject adsRecoveryObject = new AdsRecoveryObject(driver, url, DESKTOP_SIZE);
-    adsRecoveryObject.refreshPageAddingCacheBuster();
 
     adsRecoveryObject.verifyNumberOfPageFairRecoveredSlots(0);
   }
-}
 
+  private String articlePageFairRecoveryUrl() {
```

Comment:
Try to use [com.wikia.webdriver.common.core.url.UrlBuilder#appendQueryStringToURL](https://github.com/Wikia/selenium-tests/blob/346ed11190a0fbbd808406eb8dad6b433f9d7b7b/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L139) except this